README.md

```diff
@@ -2,6 +2,13 @@
 
 Boilerplate project for NodeJS, React based projects in TypeScript. This README documents whatever steps are necessary to get your application up and running.
 
+## Badge Reports
+
+| Build | Tests | Code Coverage |
+|--------|--------|--------|
+| [![Build Status](https://img.shields.io/github/actions/workflow/status/jalantechnologies/boilerplate-mern/.github/workflows/preview_on_pr_update.yml?branch=main&label=Build&job=build)] | [![Test Status](https://img.shields.io/github/actions/workflow/status/jalantechnologies/boilerplate-mern/.github/workflows/preview_on_pr_update.yml?branch=main&label=Tests&job=checks)] | [![Code Coverage](https://img.shields.io/badge/Code%20Coverage-90%25-success?style=flat)] |
```

Comment:
This always display the health of the main branch. This works for main branch.

1. But say if I have raised a PR and go to README for that branch, it should display health of that branch, is that possible?
2. When we clone a new app, how would this work - it sounds like it will always show boilerplate health, not the app health. If it cannot be automated, do we need to update README

We would also need a demo video from preview app showing this work end to end in PR description